doc news: add news for 13.0.1

HashidaTKS opened the pull request:
TODO

* [x] Translation

HashidaTKS commented:
@kou @komainu8 

Would you review this?

kou commented:
Sorry. I don't have enough time to review this for now...

HashidaTKS commented:
> Sorry. I don't have enough time to review this for now...

Sorry for bothering you. I've got it.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -362,12 +566,12 @@ msgid ""
+"OR tritonn OR \"groonga mysql\"`` と展開して欲しくなりますが、今までは、その"
+"ようなケースでは定義を追加する必要がありました。したがって、対象のキーワード"
+"に同義語がたくさんあると、同じような定義をたくさん定義する必要があるため、同"
+"義語の定義が煩雑でした。"
```
This modification does not related release note for 13.0.1.

HashidaTKS replied:
I'm thinking about diffs that automatically created by `make`...
I will revert these unrelated changes, but... should they updated?
Could we update these in another pull request?

HashidaTKS replied:
Revert changes.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -399,8 +603,8 @@ msgid ""
-"同義語を削除する際も、対象のレコードを削除するだけです。例えば、同義語から ``"
-"\"groonga mysql\"`` を削除する場合は、 ``{\"_key\": \"groonga mysql\": "
+"同義語を削除する際も、対象のレコードを削除するだけです。例えば、同義語から "
+"``\"groonga mysql\"`` を削除する場合は、 ``{\"_key\": \"groonga mysql\": "
```
ditto.

HashidaTKS replied:
Fixed.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -967,11 +1168,11 @@ msgid ""
+"Groonga\", \"Use Groonga with Ruby\"]`` は第2要素 ``Use Groonga with Ruby`` "
+"に ``Ruby`` を含むのでヒットしていますが、 ``[\"I like Ruby\", \"Use "
+"Groonga\"]`` は第2要素 ``Use Groonga`` に ``Ruby`` を含まないのでヒットしてい"
+"ません。"
```
ditto

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -1441,8 +1642,8 @@ msgid ""
-"a result, the input ``a`` was converted to ``c`` and matched to ``{\"content"
-"\":\"c\"}`` of the ``Memos`` table."
+"a result, the input ``a`` was converted to ``c`` and matched to "
+"``{\"content\":\"c\"}`` of the ``Memos`` table."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -1534,8 +1735,9 @@ msgid ""
-"\" コマンドが原因でGroongaのレスポンスが遅くなることがあります。"
+"OR検索の結果が多い場合や多くのOR条件をもつクエリー場合に、 "
+"\"request_cancel\" コマンドが原因でGroongaのレスポンスが遅くなることがありま"
+"す。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -2183,8 +2385,8 @@ msgstr ""
-"extract only text in the target sentence. Note that you need to escape ``"
-"\\`` in string."
+"extract only text in the target sentence. Note that you need to escape "
+"``\\`` in string."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -2649,8 +2851,8 @@ msgid ""
-"きます。加えて、 ``*NPP\"(a b) (1 2 3) (x y z)\"`` は、 ``'*NP\"...\" OR *NP"
-"\"...\" OR ...'`` より高速です。"
+"きます。加えて、 ``*NPP\"(a b) (1 2 3) (x y z)\"`` は、 ``'*NP\"...\" OR "
+"*NP\"...\" OR ...'`` より高速です。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -3156,8 +3358,8 @@ msgid ""
-"今回の変更によって、 ``query_parallel_or(\"tags || tags2\", \"beginner man"
-"\")`` を使った場合であっても、上記のレコードがマッチするようになりました。"
+"今回の変更によって、 ``query_parallel_or(\"tags || tags2\", \"beginner "
+"man\")`` を使った場合であっても、上記のレコードがマッチするようになりました。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -3589,11 +3791,11 @@ msgstr ""
-"following condition(``'content @ \"Groonga\" *~2.5 content @ \"Mroonga"
-"\")'`` )."
+"following condition(``'content @ \"Groonga\" *~2.5 content @ "
+"\"Mroonga\")'`` )."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -3589,11 +3791,11 @@ msgstr ""
-"具体的には、 以下の条件( ``'content @ \"Groonga\" *~2.5 content @ \"MySQL"
-"\"'`` ) にマッチしたレコードのスコアーを調整します。"
+"具体的には、 以下の条件( ``'content @ \"Groonga\" *~2.5 content @ "
+"\"MySQL\"'`` ) にマッチしたレコードのスコアーを調整します。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -3626,9 +3828,9 @@ msgstr ""
-"minus sign of ``${score_quantity}`` is reverse."
+"We can specify score quantity by ``*<${score_quantity}`` and "
+"``*>${score_quantity}``. Then, if we specify ``*<${score_quantity}``, a plus "
+"or minus sign of ``${score_quantity}`` is reverse."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -3942,8 +4144,9 @@ msgstr ""
-"\", true)'``. [Github#mroonga/mroonga#399][Reported by MASUDA Kazuhiro]"
+"'TokenFilterStopWord(\"column\", \"ignore\"), "
+"TokenFilterNFKC130(\"unify_kana\", true)'``. [Github#mroonga/mroonga#399]"
+"[Reported by MASUDA Kazuhiro]"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
+"We can customize adjustment of weight every key word by only specifying "
+"``<${WEIGHT}`` or ``>${WEIGHT}`` to target keywords since this release. For "
+"example, \"A\"'s weight is 1 and \"B\"'s weight is 0.9 in ``A <0.1B`` "
+"(\"B\"'s weight decrement 0.1)."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
-"ば、 ``A <0.1B`` では、 \"A\" の重みは1、 \"B\" の重みは0.9になります。(\"B"
-"\" の重みを0.1デクリメントしています。)"
+"ば、 ``A <0.1B`` では、 \"A\" の重みは1、 \"B\" の重みは0.9になります。"
+"(\"B\" の重みを0.1デクリメントしています。)"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4051,22 +4254,22 @@ msgstr ""
-"ただし、これらの形式(``>${WEIGHT}...``, ``<${WEIGHT}...``, and ``~"
-"${WEIGHT}...``)は互換性が無いことに注意してください。"
+"ただし、これらの形式(``>${WEIGHT}...``, ``<${WEIGHT}...``, and "
+"``~${WEIGHT}...``)は互換性が無いことに注意してください。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4090,17 +4293,17 @@ msgid ""
-"values was ``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],[\"cacao"
-"\",[\"woman\"]]``. In this release, we can get a correct reference "
+"values was ``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],"
+"[\"cacao\",[\"woman\"]]``. In this release, we can get a correct reference "
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4090,17 +4293,17 @@ msgid ""
-"先の正しい値を取得できます。"
+"``[\"apple\",[\"man\",\"many\"]],[\"banana\",[\"man\"]],[\"cacao\","
+"[\"woman\"]]`` でした。今回のリリースから、以下のように、インデックスカラム経"
+"由で参照先の正しい値を取得できます。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4131,8 +4334,8 @@ msgstr ""
-"For example, we can load data like ``\"key\" : \"[ { \"key\" : \"value"
-"\", ..., \"key\" : \"value\" } ]\"`` as below."
+"For example, we can load data like ``\"key\" : \"[ { \"key\" : "
+"\"value\", ..., \"key\" : \"value\" } ]\"`` as below."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4311,12 +4514,12 @@ msgid ""
-"今までの結果は以下のようになっていました。 ``{\"product\": \"apple\",  \"tag"
-"\": \"man\"}`` が出力されていないことがわかります。"
+"今までの結果は以下のようになっていました。 ``{\"product\": \"apple\",  "
+"\"tag\": \"man\"}`` が出力されていないことがわかります。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -4311,12 +4514,12 @@ msgid ""
-"The result will be as follows from this release. We can see that ``{\"product"
-"\": \"apple\",  \"tag\": \"man\"}`` is output."
+"The result will be as follows from this release. We can see that "
+"``{\"product\": \"apple\",  \"tag\": \"man\"}`` is output."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -7155,8 +7358,8 @@ msgstr ""
-"[:doc:`reference/executables/groonga-httpd`] Added support for ``"
-"$request_time`` in log."
+"[:doc:`reference/executables/groonga-httpd`] Added support for "
+"``$request_time`` in log."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -7176,8 +7379,8 @@ msgstr ""
-"[:doc:`reference/executables/groonga-httpd`] Added how to set the ``"
-"$request_time`` in the document."
+"[:doc:`reference/executables/groonga-httpd`] Added how to set the "
+"``$request_time`` in the document."
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -9350,13 +9553,13 @@ msgid ""
-"CP932での「U+8868 CJK UNIFIED IDEOGRAPH-8868」のように0x5c （ASCIIでの ``"
-"\\`` )を含む場合に問題がありました。"
+"CP932での「U+8868 CJK UNIFIED IDEOGRAPH-8868」のように0x5c （ASCIIでの "
+"``\\`` )を含む場合に問題がありました。"
```
ditto.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -9350,13 +9553,13 @@ msgid ""
-"length. It's caused for \"PARENTHESIZED IDEOGRAPH\" characters such as \"U"
-"+3231 PARENTHESIZED IDEOGRAPH STOCK\"."
+"length. It's caused for \"PARENTHESIZED IDEOGRAPH\" characters such as "
+"\"U+3231 PARENTHESIZED IDEOGRAPH STOCK\"."
```
ditto.

HashidaTKS replied:
Fixed.

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -10462,8 +10665,8 @@ msgid "Release 7.0.6 - 2017-08-29"
-"Supported prefix match search using multiple indexes. (e.g. ``--query \"Foo*"
-"\" --match_columns \"TITLE_INDEX_COLUMN||BODY_INDEX_COLUMN\"``)."
+"Supported prefix match search using multiple indexes. (e.g. ``--query "
+"\"Foo*\" --match_columns \"TITLE_INDEX_COLUMN||BODY_INDEX_COLUMN\"``)."
```
ditto.

komainu8 commented on doc/source/news.rst:
```diff
@@ -5,6 +5,408 @@
+       column_create --table idx_animals_sound --name animals_sound --flags COLUMN_INDEX|WITH_POSITION --type Animals --source sound
+
+       select --table Animals --query sound:@メエメエ
+       #[[0,1677829950.652696,0.01971983909606934],[[[1],[["_id","UInt32"],["_key","ShortText"],["name","ShortText"],["sound","ShortText"]],[1,"1","羊","メーメー"]]]]
```
```suggestion
       #[
           [
             0,
             1677829950.652696,
             0.01971983909606934
           ],
           [
             [
               [
                 1
               ],
               [
                 [
                   "_id",
                   "UInt32"
                 ],
                 [
                   "_key",
                   "ShortText"
                 ],
                 [
                   "name",
                   "ShortText"
                 ],
                 [
                   "sound",
                   "ShortText"
                 ]
               ],
               [
                 1,
                 "1",
                 "羊",
                 "メーメー"
               ]
             ]
           ]
         ]
```

komainu8 commented on doc/source/news.rst:
```diff
@@ -5,6 +5,408 @@
+       column_create --table idx_animals_sound --name animals_sound --flags COLUMN_INDEX|WITH_POSITION --type Animals --source sound
+
+       select --table Animals --query sound:@メエメエ
+       #[[0,1677829950.652696,0.01971983909606934],[[[1],[["_id","UInt32"],["_key","ShortText"],["name","ShortText"],["sound","ShortText"]],[1,"1","羊","メ-メ-"]]]]
```
```suggestion
       #[
           [
             0,
             1677829950.652696,
             0.01971983909606934
           ],
           [
             [
               [
                 1
               ],
               [
                 [
                   "_id",
                   "UInt32"
                 ],
                 [
                   "_key",
                   "ShortText"
                 ],
                 [
                   "name",
                   "ShortText"
                 ],
                 [
                   "sound",
                   "ShortText"
                 ]
               ],
               [
                 1,
                 "1",
                 "羊",
                 "メ-メ-"
               ]
             ]
           ]
         ]
```

HashidaTKS replied:
Fixed.

komainu8 commented on doc/source/news.rst:
```diff
@@ -5,6 +5,408 @@
+
+     table_create Entries TABLE_NO_KEY
+     column_create Entries content COLUMN_SCALAR Text
+     table_create Terms TABLE_PAT_KEY ShortText   --default_tokenizer 'TokenNgram("unify_alphabet", false,                                   "unify_digit", false)'   --normalizer NormalizerNFKC121
```
```suggestion
     table_create Terms TABLE_PAT_KEY ShortText \
       --default_tokenizer 'TokenNgram("unify_alphabet", false, \
                                                           "unify_digit", false)' \
       --normalizer NormalizerNFKC150
```

komainu8 commented on doc/source/news.rst:
```diff
@@ -5,6 +5,408 @@
+     table_create Terms TABLE_PAT_KEY ShortText \
+       --default_tokenizer 'TokenNgram("unify_alphabet", false, \
+                                       "unify_digit", false)' \
+       --normalizer NormalizerNFKC121
```
```suggestion
       --normalizer NormalizerNFKC150
```

komainu8 commented on doc/source/news.rst:
```diff
@@ -5,6 +5,408 @@
+     select Entries \
+       --filter 'content *NP10,1"abc defg$"' \
+       --output_columns '_score, content'
+     #[[0,0.0,0.0],[[[0],[["_score","Int32"],["content","Text"]]]]]
```
```suggestion
     #[
         [
           0,
           0.0,
           0.0
         ],
         [
           [
             [
               0
             ],
             [
               [
                 "_score",
                 "Int32"
               ],
               [
                 "content",
                 "Text"
               ]
             ]
           ]
         ]
       ]
```

komainu8 commented on doc/locale/ja/LC_MESSAGES/news.po:
```diff
@@ -69,6 +69,213 @@ msgstr "タグ"
+"operator`] Added a new option ``${MIN_INTERVAL}`` for near-search family."
+msgstr ""
+"[:ref:`query-syntax-near-search-condition`][:ref:`script-syntax-near-search-"
+"operator`] 各近傍検索に ``${MIN_INTERVAL}`` オプションを追加しました。"
```
この機能を追加したことによって、解決できるケースを記載して方が良いかと思いました。
「オーバラップしているフレーズをスキップできるようになるので、近傍検索の検索結果のノイズを減らすことができます。」とか。

実行例も↑のケースを実際に解決している例にすると、よりメリットが伝わりやすいと思います。

HashidaTKS replied:
現在の距離の計算方法だと説明が大変だったので省略していたのですが、記載します。

komainu8 reviewed (commented):
Could you confirm my comments?

HashidaTKS commented:
@komainu8 

Thank you, I have addressed your comments.
We are planing to check normalizers, so perhaps we need to add news items.
Please wait to merge (if there is no problem) until we finish to check normalizers.

HashidaTKS commented:
...or, merge this PR as is, and create a new pull request for adding new news topics if need.

komainu8 commented:
> @komainu8
> 
> Thank you, I have addressed your comments. We are planing to check normalizers, so perhaps we need to add news items. Please wait to merge (if there is no problem) until we finish to check normalizers.

I see.
